spec/facades/ev_station_facade_spec.rb

```diff
@@ -8,12 +8,20 @@
 
     expect(evsf).to be_a(EvStationFacade)
   end
-  it 'ev_stations', :vcr do
+  it 'all_ev_stations', :vcr do
     lat = "39.7392"
     lon = "-104.9903"
     evsf = EvStationFacade.new(lat, lon)
 
     expect(evsf.all_ev_stations.count).to eq(20)
     expect(evsf.all_ev_stations.first).to be_a(EvStation)
   end
+  it 'public_ev_stations', :vcr do
+    lat = "39.7392"
+    lon = "-104.9903"
+    evsf = EvStationFacade.new(lat, lon)
+
+    expect(evsf.public_ev_stations.count).to eq(14)
+    expect(evsf.public_ev_stations.first).to be_a(EvStation)
+  end
```

Comment:
`public_ev_stations` could be removed and wherever you are making the API call - pass the option to only return public stations by default. This will clean up code. Good looking out though!